### DISCUSSION TITLE
good old NFSEN replacement and NBAR in new nfdump 1.7

### DISCUSSION THREAD

**Discussion by abs0248** [phaag/nfdump#387] [nfdump-1.7]
Hi.
First of all thanks for new features of nfdump 1.7 .The multithreading and Geo-IP lookup are greats enhancement.
I tried the nfdump 1.7 against good old NFSEN and still struggling to make it work. But fixing all problems to make the NFSEN work is out of my capabilities.is there any good replacement  for NFSEN that we can use ? 
Also one of the other great feature of nfdump 1.7 is supporting Netflow+NBAR . I couldn't find how to show the NBAR info with nfdump in the documentation .i guess that the format to print the nbar info should be like "fmt %nbnam" but the result show nothing but <no info>. i am sure the netflow exporter (router/switch) is sending the NBAR alongside the Flexible Netflow . is there a way that i can make it work or troubleshooting the issue? 
Best Regards
Adam

**Comment by phaag** [comment]
If you use nfsen, please checkout [nfsen-1.3.9](https://github.com/phaag/nfsen). As of displaying the nbar information in nfsen, the is still an issue. Checkout nfdump(1) for the `fmt` tokens to add your own customised format. A new format can be defined on the spot at the command line or in `<prefix>/etc/nfdump.conf`

**Reply by abs0248** [reply]
Thanks  for the reply. I just want to mentioned that NBAR worked without any problem  and it can show the NBAR Application Name nicely .the problem that not showing the NBAR APP-Name was my fault as i used the `-A` switch , caused the flows to aggregated and not showing the NBAR-App-Name.
